index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-section">
+        <div class="button-logo">
+            <a href="" id="logo"> My logo </a>
+        </div>
+<nav class="normal-button">
+    <img id="button" src="Icons/Vector.png" alt="button">
+</nav>
+    </header>
+    <section>
+        <p class="text">I’m Trevor Glad to see you!</p>
```

Comment:
- [ ] Kindly make sure for the main title you use `h1` as recommended by the [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :recycle: